src/schemas/686c-674/schema.js

```diff
@@ -0,0 +1,941 @@
+                properties: {
+                  fullName: fullName,
+                  dependentType: {
+                    type: 'string',
+                    enum: ['SPOUSE', 'DEPENDENT_PARENT', 'CHILD'],
+                    enumNames: ['Spouse', 'Dependent Parent', 'Child'],
+                  },
+                  childStatus: {
+                    type: 'object',
+                    properties: {
+                      childUnder18: {
+                        $ref: '#/definitions/genericTrueFalse',
+                      },
+                      stepChild: {
+                        $ref: '#/definitions/genericTrueFalse',
+                      },
+                      adopted: {
+                        $ref: '#/definitions/genericTrueFalse',
+                      },
+                      disabled: {
+                        $ref: '#/definitions/genericTrueFalse',
+                      },
+                      childOver18InSchool: {
+                        $ref: '#/definitions/genericTrueFalse',
+                      },
+                    },
+                  },
+                },
+              },
+            },
+          },
+        },
+      },
+
+      dependentAdditionalInformation: {
+        type: 'object',
+        properties: {
+          deaths: {
+            type: 'array',
+            minItems: 1,
+            items: {
+              type: 'object',
+              properties: {
+                deceasedDateOfDeath: {
+                  $ref: '#/definitions/date',
+                },
+                deceasedLocationOfDeath: {
+                  $ref: '#/definitions/genericLocation',
+                },
+              },
+            },
+          },
+        },
+      },
+    },
+
+    reportChildMarriage: {
+      type: 'object',
+      properties: {
+        marriedChildName: fullName,
+        dateChildMarried: {
+          $ref: '#/definitions/date',
+        },
+      },
+    },
+
+    reportChildStoppedAttendingSchool: {
+      type: 'object',
+      properties: {
+        childNoLongerAtSchoolName: fullName,
+        dateChildLeftSchool: {
+          $ref: '#/definitions/date',
+        },
+      },
+    },
+
+    reportStepchildNotInHousehold: {
+      type: 'object',
+      properties: {
+        stepchildren: {
+          type: 'object',
+          properties: {
+            stepChildren: {
+              type: 'array',
+              minItems: 1,
+              items: {
+                type: 'object',
+                properties: {
+                  first: {
+                    $ref: '#/definitions/genericTextInput',
+                  },
+                  middle: {
+                    $ref: '#/definitions/genericTextInput',
+                  },
+                  last: {
+                    $ref: '#/definitions/genericTextInput',
+                  },
+                  suffix: {
+                    type: 'string',
+                    enum: suffixes,
+                  },
```

Comment:
Should this just make use of the `fullName` import like some of the other name properties?

Reply:
There are quite a few changes like this that I ignored simply because it is more of a refactor, we should do this in a separate ticket.